tests/e2e_tests.py

```diff
@@ -0,0 +1,364 @@
+import os.path
+from models.output import Output
+from generators.model_generator import ModelGenerator
```

Comment:
read about imports in pep 8, and correct them accordingly (in all files)
if you use pycharm use ctrl alt o (and ctrl alt l, theyre both your bestfriends)